app/__init__.py

```
@@ -1,137 +1,126 @@
+)
+
+# Utility functions from core
+from core.utils import resolve_attr
+from core.menu_engine import make_weekly_menu
+from core.recommendations import build_nutrition_targets
+
+# Visualization flags (optional - from bmi_visualization)
+MATPLOTLIB_AVAILABLE: Optional[bool] = False
+generate_bmi_visualization: Optional[Any] = None
+try:
+    from bmi_visualization import MATPLOTLIB_AVAILABLE  # type: ignore[no-redef]
```

Flake8 (7.3.0)

[error] 'bmi_visualization.MATPLOTLIB_AVAILABLE' imported but unused

(F401)